### PR TITLE
Include 'destroy' command

### PR DESCRIPTION
**Pull request by trsudarshan**
http://www.eclipse.org/che/docs/chedir/up-and-down/index.html documents use of the 'destroy' command which isn't documented on this page, hence added it.

### CODE REVIEW

**Comment by slemeur** [comment]
Hi @trsudarshan ! Thanks for the PR.

Before we can merge your PR, we need you to sign the Eclipse Contributor Licence Agreement. If you have not yet signed it, you can check the [following page](https://github.com/eclipse/che/wiki/Contributor-License-Agreement)
Also, we'll need your commit to be signed-off.

[More info](https://github.com/eclipse/che/wiki/How-To-Contribute#fix-bugs-or-make-enhancements)

Once the CLA is valid, will merge the PR asap.

Thanks again

**Comment by trsudarshan** [comment]
Thanks Stévan - I signed the ECA right after requesting this PR (confirmed this now via my Eclipse profile page). Besides a review, is there any thing else I would need to do to have this commit 'signed-off' upon ?

**Review by slemeur — APPROVED** [review]

**Comment by slemeur** [comment]
Hi @trsudarshan ! Sorry for my late answer. 

Yes, you would need rework your commit in order to sign-off it. When you commit, you should do a signed off commit by using the following command:
- `git commit -s`
- or `git commit --signoff`
- or add `Signed-off-by: Your Name <your.email@example.com>` at the last line of your commit message (separated with a blank line from your commit body)

(Now that you pushed the changes to your branch, you would need to rework the branch history - if it's simpler, you can open another PR ;) )

**Comment by bmicklea** [comment]
@trsudarshan can you add `Signed-off-by: Your Name <your.email@example.com>` at the last line of your commit message (separated with a blank line from your commit body)

**Comment by ghost** [comment]
Closing as an outdated PR. @trsudarshan you may open a new one against master

**Pull request closed without merging**
